TombEngine/Objects/TR3/Entity/tr3_raptor.cpp

```diff
@@ -98,15 +98,15 @@ namespace TEN::Entities::Creatures::TR3
 				ItemInfo* nearestItem = nullptr;
 				float minDistance = FLT_MAX;
 
-				for (auto* activeCreature : ActiveCreatures)
+				for (auto& currentCreature : ActiveCreatures)
 				{
-					if (activeCreature->ItemNumber == NO_ITEM || activeCreature->ItemNumber == itemNumber)
+					if (currentCreature->ItemNumber == NO_ITEM || currentCreature->ItemNumber == itemNumber)
 					{
-						activeCreature++;
+						currentCreature++;
```

Comment:
Why extra pointer increase happens here? was it ever tested? I think it may not be correct, since iterator is used.

Reply:
Looks like a remnant. Will remove.